_data/blogs.yml

```diff
@@ -1,3 +1,8 @@
+- url: https://demiprestasi.wordpress.com/2017/12/23/interview-with-a-teacher-about-fossasia-and-google-code-in/
+  title: Interview with a teacher about FOSSASIA and Google Code-in
+  img: me1.jpg
```

Comment:
rename the image to something meaningful please and this is ready to go

Reply:
Why ? because the image for my blog named me ?
is it not allowed?

Reply:
Oh ok.